stdlib/REPL/src/latex_symbols.jl

```diff
@@ -211,27 +228,22 @@ const latex_symbols = Dict(
     "\\euro" => "€",
 
 # 732 symbols generated from unicode.xml
-    "\\textexclamdown" => "¡",
+    "\\exclamdown" => "¡",
     "\\sterling" => "£",
     "\\yen" => "¥",
-    "\\textbrokenbar" => "¦",
+    "\\brokenbar" => "¦",
     "\\S" => "§",
-    "\\textasciidieresis" => "¨",
     "\\copyright" => "©",
-    "\\textordfeminine" => "ª",
+    "\\ordfeminine" => "ª",
     "\\neg" => "¬",
     "\\circledR" => "®",
-    "\\textasciimacron" => "¯",
+    "\\highminus" => "¯", # APL "high minus", or non-combining macron above
     "\\degree" => "°",
     "\\pm" => "±",
-    "\\textasciiacute" => "´",
     "\\P" => "¶",
     "\\cdotp" => "·",
-    "\\textordmasculine" => "º",
-    "\\textonequarter" => "¼",
-    "\\textonehalf" => "½",
-    "\\textthreequarters" => "¾",
-    "\\textquestiondown" => "¿",
+    "\\ordmasculine" => "º",
+    "\\questiondown" => "¿",
     "\\AA" => "Å",
     "\\AE" => "Æ",
     "\\DH" => "Ð",
```

Comment:
Wouldn't it make sense to add `\\ETH` here, for consistency with the lower case versions?

Reply:
Ok, just making sure it wasn't an overlook. Thanks for the details.